DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -143,7 +143,39 @@ public IActionResult Tutorial(int customisationId, int sectionId, int tutorialId
         [Route("/LearningMenu/{customisationId:int}/{sectionId:int}/{tutorialId:int}/Tutorial")]
         public IActionResult ContentViewer(int customisationId, int sectionId, int tutorialId)
         {
-            var model = new ContentViewerViewModel(config);
+            var candidateId = User.GetCandidateId();
+            var centreId = User.GetCentreId();
+
+            var tutorialContent = tutorialContentService.GetTutorialContent(customisationId, sectionId, tutorialId);
+
+            if (tutorialContent?.TutorialPath == null || centreId == null)
+            {

```

Comment:
A log would be useful, as I think we have in the other actions? 🙂

Reply:
Added in b073a6a 🙂